bindata/bootkube/bootstrap-manifests/etcd-member-pod.yaml

```diff
@@ -120,6 +120,8 @@ spec:
       #!/bin/sh
       set -euo pipefail
 
+      {{ if eq .Platform "Azure"}} export ETCD_HEARTBEAT_INTERVAL="500" {{ end }}
+
```

Comment:
we also need to bump leader election timeout[1].

[1] https://github.com/openshift/cluster-etcd-operator/blob/8f7718d45c4a31235c276322365ba10432921adb/pkg/etcdenvvar/etcd_env.go#L225

Reply:
updated